RobotArmAPP/Classes/ConvertToString.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace RobotArmAPP.Classes
+{
+    class ConvertToString
+    {
+        public string ConvertItemToString(int garra, int axis4, int axis3, int axis2, int axis1, int speed, int delay)
```

Comment:
```suggestion
        public string ConvertMovementToString(Movement movement)
```
Veja como fica no seu código criar uma classe que reune todos esses itens `garra, axis1, ...`. Nesta classe, talvez imbutir esses métodos de `ToString()`. Assim, toda vez que for usar um movimento em forma de string, ficaria fácil, só jogar `String movementString = movement.ToString()`

Reply:
Feito!